docs/supervisor-datamgr.md

```diff
@@ -0,0 +1,72 @@
+# Data manager configuration for vSphere with Kubernetes
+In a vSphere with Kubernetes environment, the data manager should be installed as a Virtual Machine on vSphere. It does not come
```

Comment:
Can you add a section explaining what does Data Manager do?  When it is required?

Reply:
Done.